app/models/house_member.rb

```diff
@@ -0,0 +1,3 @@
+class HouseMember < ApplicationRecord
+
+end
```

Comment:
Leaving a note that in the future we may need to create the relationship with users through the joins table